src/context-parser-handlebars.js

```diff
@@ -894,6 +963,25 @@ ContextParserHandlebars.prototype.handleRawBlock = function(input, i, saveToBuff
     throw "Parsing error! Cannot encounter '}}}}' close brace of raw block.";
 };
 
+/**
+* @function ContextParserHandlebars.reset
+*
+* @description
+* All non-config internal variables are needed to reset!
+*/
+ContextParserHandlebars.prototype.reset = function() {
```

Comment:
i may fallback to new an instance after enhancing the express-secure-handlebars, seems there is no need to reuse the CPH